docs/getting-started.md

```diff
@@ -9,65 +9,234 @@ sidebar_label: Getting started
+
+To install it from the command line run:
 
 ```sh
 npm i -g @aragon/cli
 ```
 
-Once we have this package installed we can start building DAOs.
+Hopefully, it downloaded successfully. If that's the case, congrats! you're now officially ready to start building DAOs!
 
-### Note on Git
+If you're having trouble with this step, you should read through the relevant operating system section below. If that doesn't fix things, please don't hesitate to reach out to us at the [#dev-help channel on the Aragon Chat](https://aragon.chat/channel/dev-help).
+
+### Mac considerations
+
+If you're seeing one or more errors that look like:
+
+```sh
+EACCES: permission denied
+```
+
+It's probably because you originally installed Node with root permissions [explain what this means] . Because of this, writing to your npm directory (```npm -i -g```) requires root permissions too.
+
+While it's not a good idea to have Node installed this way, one way to quickly give yourself root permissions is to run the slightly modified command: [maybe not a good idea to show this option]
+
+```sh
+sudo npm i -g --unsafe-perm @aragon/cli
+```
+
+An arguably better way to fix the problem is to follow the steps outlined in this [stackoverflow answer.](https://stackoverflow.com/a/24404451) [should make this more intuitive...]
 
-You might need to have [Git](https://git-scm.com) installed.
+[i think this whole section needs to be exlained more simply...]
 
 ### Windows considerations
 
-You might need to run the shell with administrator rights when installing the aragonCLI, because our `go-ipfs` dependency will need to create a symlink to work correctly.
+You might need to run the shell with administrator rights when installing the aragonCLI, because our `go-ipfs` dependency will need to create a symlink to work correctly. [this needs to be explained better]
 
-If you have problems during the instalation of aragonCLI or any other dependencies. You probably need to install [windows-build-tools](https://www.npmjs.com/package/windows-build-tools) or similar package.
+If you have problems during the instalation of aragonCLI or any other dependencies. You probably need to install [windows-build-tools](https://www.npmjs.com/package/windows-build-tools) or similar package. 
 
+Again, if you're having trouble fixing things, please reach out to us at the [#dev-help channel on the Aragon Chat](https://aragon.chat/channel/dev-help)
+
+### Note on Git
+
+You might need to have [Git](https://git-scm.com) installed. [a little startling for the reader... why might? i think we need to elaborate here...]
 
 ## Quick start
 
-To create your first DAO run:
+In order to get up and running quickly, we’ll build our first DAO using some basic scaffolding [explain what scaffolding means] -- remember a DAO is just a combination of Aragon Apps... [elaborate...]
+
+To create our DAO run:
 
 ```sh
 npx create-aragon-app foo.aragonpm.eth
```

Comment:
We could change `foo` to something more related like `starting` or `first_app`

Reply:
How about ```first_dao```?

Reply:
Went with ```first_dao```. Let me know if that's ok or if you prefer something else.

Reply:
I like `first_dao`